ArchUnitNETIntegrationTests/PlantUmlDiagramsFromDependenciesTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ArchUnitNET.Domain;
+using ArchUnitNET.Fluent;
+using ArchUnitNET.Fluent.PlantUml;
+using ArchUnitNET.Fluent.Slices;
+using ArchUnitNET.Loader;
+using ArchUnitNET.MSTestV2;
+using Xunit;
+
+namespace ArchUnitNETIntegrationTests;
+
+public class PlantUmlIntegrationDiagramTests
+{
+    [Fact]
+    public void ComponentDiagramFromSlicesBuild()
+    {
+        var sliceRule1 = SliceRuleDefinition.Slices().Matching("Microsoft.VisualStudio.TestPlatform.(**).");
+        var arch1 = new ArchLoader()
+            .LoadAssembly(typeof(Microsoft.VisualStudio.TestPlatform.TestSDKAutoGeneratedCode).Assembly).Build();
+        var path1 = "../../../PlantUmlDiagramsFromDependencies/Microsoft.VisualStudio.TestPlatform.puml";
+        PlantUmlDefinition.ComponentDiagram().WithDependenciesFromSlices(sliceRule1, arch1).WriteToFile(path1);
+        Assert.True(File.Exists(path1));
```

Comment:
I'm a bit confused here. Is the purpose of this test to generate the puml files that you checked in?